### ISSUE TITLE
The store.update function works incorrectly when the $set fields contain only null values.

### ISSUE THREAD

**Issue by Matosov** [reimagined/resolve#1758]
For instance the following update 
await store.update('Tickets', { id: aggregateId }, { $set: { contactId: null, contactName: null } });
leads to the error:
[{"code": "BadRequestException", "name": "BadRequestException", "stack": "BadRequestException: ERROR: syntax error at or near \"WHERE\"\n  Position: 58\n    at Object.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/json.js:52:27)\n    at Request.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/rest_json.js:55:8)\n    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\n    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/request.js:688:14)\n    at Request.transition (/var/runtime/node_modules/aws-sdk/lib/request.js:22:10)\n    at AcceptorStateMachine.runTo (/var/runtime/node_modules/aws-sdk/lib/state_machine.js:14:12)\n    at /var/runtime/node_modules/aws-sdk/lib/state_machine.js:26:10\n    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:38:9)\n    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:690:12)", "message": "ERROR: syntax error at or near \"WHERE\"\n  Position: 58"}].

**Comment by MrCheater** [comment]
```js  
await store.update('Tickets', {​​​​​​​​ id: aggregateId }​​​​​​​​, {​​​​​​​​ $set: {​​​​​​​​ contactId:null, contactName:null }​​​​​​​​ }​​​​​​​​)
```
->

```
UPDATE "readmodel-prod-0.28.2-xxxxxx"."Tickets"
    SET  WHERE 
    ((("id" = CAST('"e3370191-9db0-449a-bf50-e7b0d45cbb55"' AS JSONB)) and (not ("id" is null)) and (not (CAST('"e3370191-9db0-449a-bf50-e7b0d45cbb55"' AS JSONB) is null))) or      
    (("id" is null) and (CAST('"e3370191-9db0-449a-bf50-e7b0d45cbb55"' AS JSONB) is null)))
   ;
```

**Comment by MrCheater** [comment]
Fixed in version 0.29.0
